src/proxy/outbound.rs

```diff
@@ -136,6 +159,33 @@ pub struct OutboundConnection {
     pub hbone_port: u16,
     pub metrics: Arc<Metrics>,
     pub id: TraceParent,
+    pub pool: Pool<PoolValue, PoolKey>,
+}
+
+#[derive(Debug, Clone)]
+pub struct PoolClient(http2::SendRequest<Body>);
+
+impl Poolable for PoolClient {
+    fn is_open(&self) -> bool {
+        // TODO
```

Comment:
What's the implication of always giving true? Just that something always goes back into the pool?

Reply:
IDK, that is the main thing left to do for this PR aside from waiting for upstream